pyproject.toml

```diff
@@ -83,4 +83,5 @@ max-complexity = 25
 dev = [
     "pytest>=7,<8",
     "python-dotenv>=1,<2",
+    "cryptography>=43.0.0"
```

Comment:
You added this to the wrong section. Move it to line 15, keep them in alphabetical order.

Reply:
Oops, fixed!